real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -100,6 +111,24 @@ public static void loadConfigFile() {
             setPortVision(configJson.get("PORT_VISION").getAsString());
             setWayPointSourceLogFile(configJson.get("WAYPOINT_SOURCE_LOG_FILE").getAsString());
             setPlayBackSourceFile(configJson.get("PLAY_BACK_SOURCE_FILE").getAsString());
+            RobobuggyConfigFile.robotType = configJson.get("ROBOT_TYPE").getAsString();
+            switch(robotType){
```

Comment:
Being nit picky here: Why RobobuggyConfigFile.roboType on 114 and only robotType everywhere else?